.circleci/config.yml

```diff
@@ -0,0 +1,89 @@
+version: 2
+gpu: &gpu
+  machine:
+    image: ubuntu-1604:201903-01
+  resource_class: gpu.medium
+
+jobs:
+  build_install_test_ubuntu:
+    <<: *gpu
+    steps:
+      - checkout:
+          path: ./habitat-sim
+      - run:
+          name: Install cmake
+          no_output_timeout: 60m
+          command: |
+              wget https://github.com/Kitware/CMake/releases/download/v3.13.4/cmake-3.13.4-Linux-x86_64.sh
+              sudo mkdir /opt/cmake
+              sudo sh ./cmake-3.13.4-Linux-x86_64.sh --prefix=/opt/cmake --skip-license
+              sudo ln -s /opt/cmake/bin/cmake /usr/local/bin/cmake
+      - run:
+          name: Install conda
+          no_output_timeout: 60m
+          command: |
+              curl -o ~/miniconda.sh -O  https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh
+              chmod +x ~/miniconda.sh
+              ~/miniconda.sh -b -p $HOME/miniconda
+              rm ~/miniconda.sh
+              $HOME/miniconda/bin/conda install -y numpy pyyaml scipy ipython mkl mkl-include
+              export PATH=$HOME/miniconda/bin:$PATH
+              conda create -y -n habitat python=3.6
+              . activate habitat
+              conda install -y numpy pyyaml scipy ipython mkl mkl-include pytest
```

Comment:
Let's add a `conda install -y -c conda-forge ninja` -- This should help with build speed